api-example.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
+        link href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
```

Comment:
@e-kolpakov Garh, I was sloppy at copy-pasting. Not sure how I missed that when looking at the diff, thanks for noticing.